includes/discovery/storage/hrstorage.inc.php

```diff
@@ -43,6 +43,10 @@
             continue;
         }
 
+        if ($device['os'] == 'aix') {
```

Comment:
Just change this to:

```php
if ($device['os'] == 'aix' && !empty($aix_filesystem)) {
```

Reply:
Also, a comment would help, "Skip hrStorage if aixFsTable is available".

Reply:
Thank you!